src/test/java/com/cag/cagbackendapi/service/organization/OrganizationServiceTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.cag.cagbackendapi.service.organization
+
+import com.nhaarman.mockito_kotlin.*
+import org.mockito.Mock
+import org.mockito.InjectMocks
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.junit.jupiter.MockitoExtension
+import org.slf4j.Logger
+
+@ExtendWith(MockitoExtension::class)
+internal class OrganizationServiceTest {
+
+    @Mock
+    private lateinit var logger: Logger
```

Comment:
How do you know when you will need a Lateinit variable? Is it something that should be used often?

Reply:
Because this class is a mock we need to use lateinit var because Spring needs to create a bean for this class. We don't use lateinit var very much, it's normally just for testing when we want to mock the services we'll test and in some of the spring bean configuration for the project. There is a lot to learn about Spring Framework and how it works to understand how beans are special and need stuff like this. It's rare you'll need to worry about it, but we can go into more depth on this later!